Update irq_handler to match new irq_handler_t

See commit 7d12e780e003f93433d49ce78cfedf4b4c52adc5 that went in
Linux 2.6.19

Signed-off-by: Herton Ronaldo Krzesinski <herton@mandriva.com>

--- fcpci-3.11.07.orig/driver.c	2008-01-09 17:50:49.000000000 -0200
+++ fcpci-3.11.07/driver.c	2008-01-09 23:51:17.000000000 -0200
@@ -107,7 +107,7 @@ static int			card_id			= 0;
 /*---------------------------------------------------------------------------*\
 \*---------------------------------------------------------------------------*/
 static void scheduler (unsigned long data);
-static irqreturn_t irq_handler (int irq, void * args, struct pt_regs * regs);
+static irqreturn_t irq_handler (int irq, void * args);
 
 static DECLARE_TASKLET_DISABLED (scheduler_tasklet, scheduler, 0);
 
@@ -836,11 +836,10 @@ static void scheduler (unsigned long dat
 
 /*---------------------------------------------------------------------------*\
 \*---------------------------------------------------------------------------*/
-static irqreturn_t irq_handler (int irq, void * args, struct pt_regs * regs) {
+static irqreturn_t irq_handler (int irq, void * args) {
 	int	res	= IRQ_NONE;
 	
 	UNUSED_ARG (irq);
-	UNUSED_ARG (regs);
 	if (args != NULL) {
 		assert (capi_lib->cm_handle_events != NULL);
 		if (atomic_read (&scheduler_id) == smp_processor_id ()) {
